conans/client/rest/rest_client_v1.py

```diff
@@ -308,3 +309,12 @@ def get_latest_recipe_revision(self, ref):
 
     def get_latest_package_revision(self, pref):
         raise NoRestV2Available("The remote doesn't support revisions")
+
+    def _post_json(self, url, payload):
```

Comment:
is there no `_post_json` call in V2?

Reply:
Having a look at the code I wonder if this is needed. I think a `get_json()` with `data` argument should be the same. the only difference I see are the custom_headers

Reply:
No, it was not being used in V2, this is why I moved it to the place it is used (reducing unnecessary big scope).

The problem is that ``get_json()`` does more things: add headers, interprets return codes and raise exceptions. That wouldn't be a refactor, but probably change behavior, I don't want to do that here.